src/CodeMash.Repository/CodeMashRepository.cs

```diff
@@ -702,12 +702,26 @@ public List<TA> Aggregate<TA>(PipelineDefinition<T, TA> aggregation, AggregateOp
 
         public long Count(FilterDefinition<T> filter, CountOptions countOptions = null)
```

Comment:
I think we need to have one method extra : public long Count() where filter is not defined. then you call another method with empty filter (x => true) or FilterDefinition<T1>.Empty ...